api/Element.json

```diff
@@ -3452,28 +3463,42 @@
                 ]
               }
             ],
-            "safari": {
-              "version_added": "11.1",
-              "partial_implementation": true,
-              "notes": "Does not support the <code>subtree</code> option.",
-              "flags": [
-                {
-                  "type": "preference",
-                  "name": "Web Animations"
-                }
-              ]
-            },
-            "safari_ios": {
-              "version_added": "11.3",
-              "partial_implementation": true,
-              "notes": "Does not support the <code>subtree</code> option.",
-              "flags": [
-                {
-                  "type": "preference",
-                  "name": "Web Animations"
-                }
-              ]
-            },
+            "safari": [
+              {
+                "version_added": "13.1",
+                "partial_implementation": true,
+                "notes": "Does not support the <code>subtree</code> option."
```

Comment:
Confirmed: https://github.com/WebKit/WebKit/blob/27bbb8b26e7c86cef190b68a12127e492187e2ca/Source/WebCore/animation/DocumentOrShadowRoot%2BWebAnimations.idl#L28

Reply:
Oops, that was the wrong bit of IDL. It is actually supported:
https://github.com/WebKit/WebKit/blob/27bbb8b26e7c86cef190b68a12127e492187e2ca/Source/WebCore/animation/Animatable.idl#L29

I'll send a PR to fix.